types/react-fa/index.d.ts

```diff
@@ -37,7 +37,6 @@ export const Icon: ComponentClass<IconProps>;
 
 export interface IconStackProps extends SizeOverrideHTMLProps<IconStack> {
     size?: IconSize;
-    children?: IconProps[];
```

Comment:
the `children` prop is not type checked in TSX blocks, unless it is written as an attribute, which doesn't even work properly because TSX blocks always type to `JSX.Element` not their react component type.

For example, if we decided to make a component that must have a `div` as its child, typescript will not prevent a `span` from passing type checks.
```ts
interface FooProps {
  children?: HTMLDivElement[];
}

class Foo extends React.Component<FooProps, {}> {
}

const bar = (
  <Foo>
    <span />
  </Foo>
)
```